README.md

```diff
@@ -111,16 +111,42 @@ Hello World!
 
 Stargz snapshotter also supports [further configuration](/docs/overview.md) including private registry authentication, mirror registries, etc.
 
-## Creating eStargz images with optimization
+## Getting eStargz images
 
 - For more examples and details about the image converter `ctr-remote`, refer to [Optimize Images with `ctr-remote image optimize`](/docs/ctr-remote.md).
 - For more details about eStargz format, refer to [eStargz: Standard-Compatible Extensions to Tar.gz Layers for Lazy Pulling Container Images](/docs/stargz-estargz.md)
 
 For lazy pulling images, you need to prepare eStargz images first.
-You can use [`ctr-remote`](/docs/ctr-remote.md) command for do this.
-You can also try our pre-converted images listed in [Trying pre-converted images](/docs/pre-converted-images.md).
+There are several ways to achieve that.
+This section describes some of them.
 
-In this section, we introduce `ctr-remote` command for converting images into eStargz with optimization for reading files.
+### Trying pre-built eStargz images
+
+You can try our pre-converted eStargz images on ghcr.io listed in [Trying pre-converted images](/docs/pre-converted-images.md).
+
+### Registry-side conversion with `estargz.kontain.me`
+
+You can convert arbitrary images into eStargz on the registry-side, using [`estargz.kontain.me`](https://github.com/imjasonh/kontain.me/tree/012b357a5e8f4a301e929c5739a4905b0e7bad86/cmd/estargz).
```

Comment:
You can link to https://estargz.kontain.me and it'll redirect to the latest README.md, or theoretically in the future some other better landing page.

Reply:
Thank you for sharing the link! fixed docs to use that.